README.md

```diff
@@ -21,6 +21,28 @@ If you need support or want to help developing please visit the [official wiki](
 [![Gitter chat](https://badges.gitter.im/InvoicePlane/InvoicePlane.png)](https://gitter.im/InvoicePlane/InvoicePlane)   
 [![Freenode](https://invoiceplane.com/content/badges/badge_freenode.png)](irc://irc.freenode.net/InvoicePlane)
 
+### Developing with Vagrant
```

Comment:
Please move this to a new file called CONTRIBUTION.md and extend your explanations.  "Be sure to also download VirtualBox Extensions!" is not very precise for a beginner.